src/constants.js

```diff
@@ -236,3 +236,6 @@ export const UNAUTHORIZED_CODE = 403;
 export const VERSION_RESTORE_ACTION = 'restore';
 export const VERSION_DELETE_ACTION = 'delete';
 export const VERSION_UPLOAD_ACTION = 'upload';
+
+/* ------------------ Placeholder Feed Items ------------------------- */
+export const PLACEHOLDER_USER = { type: 'user', id: '2', name: '' };
```

Comment:
Can we default this without adding this user? There may be cases where it's not actually an anonymous user so this data would be inaccurate.

Reply:
As discussed, leaving as is.